enhancements/cloud-integration/infrastructure-external-platform-type.md

```diff
@@ -0,0 +1,560 @@
+---
+title: infrastructure-external-platform-type
+authors:
+  - "@lobziik"
+  - "@elmiko"
+reviewers:
+  - "@dhellmann"
+  - "@mhrivnak"
+  - "@rvanderp3"
+  - "@mtulio"
+  - "@deads2k, to review library-go, KCMO related parts andopenshift/api changes" 
+  - "@JoelSpeed, to review CCM, MAPO related parts and openshift/api changes"
+  - "@sinnykumari, to review MCO related parts"
+  - "@danwinship, to review CNO related parts"
+  - "@Miciah, to review Ingress related parts"
+  - "@openshift/openshift-team-windows-containers"
+approvers:
+  - "@dhellmann"
+  - "@deads2k"
+  - "@bparees"
+api-approvers:
+  - "@deads2k"
+  - "@JoelSpeed"
+creation-date: 2022-09-06
+last-updated: 2022-09-23
+tracking-link:
+  - https://issues.redhat.com/browse/OCPPLAN-9429
+  - https://issues.redhat.com/browse/OCPPLAN-8156
+see-also:
+  - "[KEP-2392 Cloud Controller Manager](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cloud-provider/2392-cloud-controller-manager)"
+  - "[OCP infrastructure provider onboarding guide](https://docs.providers.openshift.org/overview/)"
+  - "[Out-of-tree cloud provider integration support](https://github.com/openshift/enhancements/blob/master/enhancements/cloud-integration/out-of-tree-provider-support.md)"
+  - "[Platform Operators Proposal](https://github.com/openshift/enhancements/blob/master/enhancements/olm/platform-operators.md)"
+  - "[Capabilites selection](https://github.com/openshift/enhancements/blob/master/enhancements/installer/component-selection.md)"
+  - "[Bare metal in-cluster network infrastructure](https://github.com/openshift/enhancements/blob/ce4d303db807622687159eb9d3248285a003fabb/enhancements/network/baremetal-networking.md)"
+---
+
+# Introduce new platform type "External" in the OpenShift specific Infrastructure resource
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+In an effort to reduce the amount of time Red Hat engineers spend directly involved with third party engagements,
```

Comment:
nit: this is the win on the Red Hat side.  The win on the third-party-platform side is lower dev latency, because they don't need to wait on a Red Hat approver to unblock any development loop.  It might be worth mentioning that win/win, so it's clear that we still like third-party integrators, and are happy to talk to them when we have time ;)

Reply:
But the motivation section is down below the summary, and external folks might be bummed out reading this until they get down to the motivation section and brighten back up ;)